module3/labs/Week4/DeleteResource.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Delete Resource Lab
+---
+
+## Setup
+
+You've already had a chance to practice implementing Delete by building a delete button for reviews, so for today's lab we're going to do something a bit different than what we covered in class. 
+
+You're going to continue working on the MVC movies app and implement new features to make the application more user-friendly.
+
+## Brainstorm Features
+
+In software development, a `feature` is a specific piece of functionality that fulfills a specific user need or requirement. 
+
+We're going to start this afternoon's lab by brainstorming possible features we could build to improve our MVC movies app. Specifically, we want to think about features that would make it easier for users to navigate our application. 
+
+> Take 5 minutes to try using the MVC Movies app and write down features that would make the application more user friendly. When we come back together, your instructor will call on folks to contribute feature ideas and compile them into one big list.
+
+Here is one example of a feature idea: "When a user clicks the name of a movie in the all movies page, it takes them to the movies details page for the movie clicked"
+
+<aside class="instructor-notes" markdown="1">
+    <p><strong>Instructor Note</strong><br>I recommend sharing your screen with slack displayed and compiling a list of student feature ideas in a slack post. I would also add spicy pepper emojis next to the features that will be more difficult. Something to bring up when students are brainstorming features is the guiding light of only navigating using the page. No typing in URLs, and no typing the back button.</p>
+</aside>
+
+
+<aside class="instructor-notes" markdown="1">
+    <p><strong>Instructor Note</strong><br>Here is a particularly spicy feature request, if students are looking for a challenge. 
+    In the title it says - MVC Movies. It's supposed to have the page before that. For example, Reviews - MVCMovie. Can you figure out what is setting that?</p>
+</aside>
+
+
+## Implementation
+
+With your partner, you now have a chance to implement some of the feature requests! Before attempting any of the 🌶️ features, implement at least two regular features.
+
+## Presentations

```

Comment:
I really like this lab - I love that it builds on the lesson, and pushes students to do some independent thinking and decision making!